NSS-Day-Cohort-66/rare-api-eager-miracles
# Sn edit tag

## Edit tag Logic

* Mew method for edit Tag.

## How to Test
* pull down SN-EditTag
* Make sure you have the Api debugger running
* Open postman and make a PUT request to http://localhost:8000/tags/{insertNumber}
``` 
{
    "label": "{new name for tag}"
}
```
* Return payload 204 no content
